### ISSUE TITLE
Elevator selection doesn't work with Optifine 1.15.2

### ISSUE THREAD

**Issue by JayCeeCreates** [Thuts-Mods/ThutCore#4]
I have Optifine 1.15.2 pre30 and Sildur's Enhanced Default on, and every time I select an elevator area, this appears:
![ignore the 1 FPS, it's doing the screenshot](https://user-images.githubusercontent.com/28774264/86727536-1a5d4380-c05e-11ea-85eb-f8198b8a93cd.png)
And when I finally selected an area, this happens:
![2020-07-07_14 26 00](https://user-images.githubusercontent.com/28774264/86727986-8b9cf680-c05e-11ea-8d6c-62717317b9b2.png)
All elevators created are vanished, but can still be interacted.

In other words, elevators would render properly, but the selection area apparently doesn't. A fix would be appreciated. :)
